protobuf/lib/meta.dart

```diff
@@ -21,9 +21,6 @@ const GeneratedMessage_reservedNames = <String>[
   'clone',
   'copyWith',
   'createEmptyInstance',
-  'createMapField',
```

Comment:
Perhaps we should leave these? Otherwise the names will no longer be disambiguated breaking existing code?
Not sure what is better here...

Reply:
Do you suggest we keep them indefinitely, or remove later?

If you want to remove later, this is already a breaking change so we could do it now.

Reply:
Yeah - it is either remove 

* now or 
* never.

Not sure what is better. We could check if we have any usages inside google3. These names are probably quite rare - so maybe it is better to remove now.

Reply:
I tested the CL globally internally it doesn't break anything.

Reply:
Ok, in that case I vote for just removing them